### ISSUE TITLE
Non-selectable nodes (e.g., case branch labels) should be indicated somehow

### ISSUE THREAD

**Issue by dhess** [hackworthltd/primer-app#59]
Here the `True` and `False` labels on the `case` branches are not selectable:

<img width="303" alt="Screen Shot 2021-01-27 at 3 27 00 PM" src="https://user-images.githubusercontent.com/2438/106013165-1b19fd80-60b4-11eb-8bf6-b313745ec697.png">

We should visually indicate that restriction to the user somehow.

**Comment by georgefst** [comment]
> I would rather address this by making everything in the tree selectable and "actionable," even if you can't perform an "action" on it (in the Vonnegut sense of "action").

Agreed. I'm pretty sure that the only things which are not selectable now are constructors and wildcards in patterns. Plus pattern boxes themselves, if you consider those to count (they are nodes as far as the API is concerned). This can be seen by the places in `Primer.API` where we have to cook up non-numeric IDs.
